Source/App/EbAppInputy4m.c

```diff
@@ -6,7 +6,12 @@
 #include "EbAppInputy4m.h"
 #define YFM_HEADER_MAX 80
 #define YUV4MPEG2_IND_SIZE 9
-#define PRINT_HEADER 0
+
+// Define PRINT_HEADER to log y4m header contents
+#ifndef NDEBUG
+#define PRINT_HEADER
+#endif
+
```

Comment:
Hi, even a debug build, we don't really need to print these logs. They are not something special that we need to pay attention in debug session. Please turn if off too for debug build.

Reply:
I would argue that it would have a value in debug mode when you are trying to confirm what the encoder sees when reading y4m files to make sure there isn't a problem with the y4m video nor in the y4m code

Reply:
If for this reason, there will so many more logs to print. 
To validate y4m, ffprobe can be used. 
And it is easy to debug those printed values in VS if anything wrong in y4m code.

Reply:
I guess

Reply:
I will turn off in debug builds.  Note this was in the AV1 implementation I ported this feature from.

Reply:
Thx Mark. AV1 is still under dev. Clean up is needed.

Reply:
Just to be clear...should I remove the PRINT_HEADER feature all together, or just disable in debug/release builds (e.g. #undef PRINT_HEADER). imho it's useful when debugging

Reply:
Pls remove this line: #define PRINT_HEADER 0